custom-namespaces.yaml

```diff
@@ -414,6 +414,11 @@ firefox_desktop:
       type: growth_accounting_explore
       views:
         base_view: growth_accounting
+    events:
+      type: events_explore
+      views:
+        base_view: events
+        extended_view: desktop_events_table
```

Comment:
lookml-generator automatically adds `events_unnested` to `extended_views`, however for firefox_desktop we use `desktop_events_table` which has legacy telemetry data